common/app/views/fragments/amp/header.scala.html

```diff
@@ -16,13 +16,17 @@ <h1 class="u-h">The Guardian</h1>
         </a>
 
         <nav class="header__nav">
+            <ul class="header__nav__primary-links">
             @NewNavigation.PrimaryLinks.map { link =>
-                <a class="header__nav__link"
-                    href="@LinkTo(link.url)"
-                    data-link-name="amp : nav : @link.title">
-                        @link.title
-                </a>
+                <li class="header__nav__link-item">
```

Comment:
In BEM there is only one level of nesting allowed :) So this should better be `header__nav-link-item`. Applies to a few cases following aswell.